exetera/core/dataframe.py

```diff
@@ -530,8 +530,31 @@ def to_csv(self, filepath:str, row_filter:Union[np.ndarray, fld.Field]=None, col
                 else:
                     start_row += chunk_row_size
 
-            
-    def drop_duplicates(self, by: Union[str, List[str]], 
+    def to_pandas(self, row_filter=None, col_filter: Union[str, List[str]] = None):
+        """
+        Convert an ExeTera dataframe to Pandas DataFrame.
+        :param row_filter:
```

Comment:
Need to explain what these parameters are, could a type for `row_filter` be provided?

Reply:
updated.